TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -9,14 +9,27 @@
 using TypeCobol.Compiler.Nodes;
 using TypeCobol.Compiler.Parser;
 using System.Text.RegularExpressions;
+using TypeCobol.Compiler.Directives;
 using TypeCobol.Compiler.Scanner;
 
 namespace TypeCobol.Compiler.Diagnostics
 {
     public class CrossCompleteChecker : AbstractAstVisitor
     {
+        public CrossCompleteChecker(TypeCobolOptions compilerOptions = null)
+        {
+            if (compilerOptions == null)
+                CompilerOptions = new TypeCobolOptions();
+            else
+                CompilerOptions = compilerOptions;
+            _checkEndAlignment = CompilerOptions.CheckEndAlignment;
+        }
+        private TypeCobolOptions CompilerOptions { get; }
+
         private Node CurrentNode { get; set; }
 
+        private readonly bool _checkEndAlignment;
```

Comment:
Either keep _checkEndAlignment or CompilerOptions but not both. I think `private readonly TypeCobolOptions _compilerOptions;` is best.

Reply:
fixed
https://github.com/TypeCobolTeam/TypeCobol/tree/828d955ed380f37e3a26def80e2d534f11fb26d7